src/spatial/2d.js

```diff
@@ -240,8 +239,15 @@ Crafty.c("2D", {
             },
             configurable: true,
             enumerable: true
-        });
-        Object.defineProperty(this, '_visible', {enumerable:false});
+        },
+        _visible: {enumerable:false}
+
+    },
+
+    _define2DProperties: function () {
+        for (var prop in this._2D_property_definitions){
```

Comment:
does this only iterate over the properties above, or also over the properties in the property chain, if there are any?